src/v/reflection/adl.h

```diff
@@ -45,7 +45,10 @@ struct adl {
     static constexpr bool is_chrono_milliseconds
       = std::is_same_v<type, std::chrono::milliseconds>;
     static constexpr bool is_time_point
-      = std::is_same_v<type, ss::lowres_system_clock::time_point>;
+      = std::is_same_v<
+          type,
+          ss::lowres_system_clock::
+            time_point> || std::is_same_v<type, ss::lowres_clock::time_point>;
```

Comment:
`ss::lowres_clock` doesn't correspond to wall-clock time. It should not be serialised.